engine/flink/components/kafka/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/FlinkKafkaComponentProvider.scala

```diff
@@ -39,78 +28,11 @@ class FlinkKafkaComponentProvider extends ComponentProvider {
     val overriddenDependencies = TemporaryKafkaConfigMapping.prepareDependencies(config, dependencies)
     val docsConfig: DocsConfig = DocsConfig(config)
     import docsConfig._
-    val avro                    = "DataSourcesAndSinks#schema-registry--avro-serialization"
-    def universal(typ: String)  = s"DataSourcesAndSinks#kafka-$typ"
-    val schemaRegistryTypedJson = "DataSourcesAndSinks#schema-registry--json-serialization"
-    val noTypeInfo              = "DataSourcesAndSinks#no-type-information--json-serialization"
+    def universal(typ: String) = s"DataSourcesAndSinks#kafka-$typ"
```

Comment:
```suggestion
    def universal(processingType: String) = s"DataSourcesAndSinks#kafka-$processingType"
```

Reply:
It isn't a processing type - it's a component type. I changed it.